schemas/notes.py

```diff
@@ -1,10 +1,16 @@
+from ma import ma
 from models.user import UserModel
 from models.tickets import TicketModel
+from models.notes import NotesModel
 from utils.time import time_format
-from marshmallow import fields, Schema, validates, ValidationError
+from marshmallow import fields, validates, ValidationError, post_load, EXCLUDE
 
 
-class NotesSchema(Schema):
+class NotesSchema(ma.SQLAlchemyAutoSchema):
```

Comment:
Originally we were not using an autoschema here and explicitly defining the field attributes. If we're going to switch this to an auto schema then I don't think we need all of these explicitly defined field attributes below.